gensim/corpora/textcorpus.py

```diff
@@ -552,7 +554,7 @@ def getstream(self):
         """
         for path in self.iter_filepaths():
             logging.debug("reading file: %s", path)
-            with utils.smart_open(path) as f:
+            with utils.smart_open(path, 'rt') as f:
```

Comment:
This looks fragile. Best to always open files in binary mode `rb`, and convert to text (unicode) explicitly, with an explicit encoding, where needed.